setup.cfg

```diff
@@ -93,6 +28,11 @@ known_third_party = platformdirs, astroid, sphinx, isort, pytest, mccabe, six, t
 skip_glob = tests/functional/**,tests/input/**,tests/extensions/data/**,tests/regrtest_data/**,tests/data/**,astroid/**,venv/**
 src_paths = pylint
 
+[flake8]
```

Comment:
Can't this go in `.flake8`?

Reply:
Let's do it the other way round and move the whole flake8 config into `setup.cfg`. In the past, we have also removed `.isort`. At some point we can move most / all of these configs into `pyproject.toml`.

Reply:
Strong agree. I don't think ``.flake8`` support pyproject.toml though right now though . But there's other tool we could move already, tox and black probably.

Reply:
(https://github.com/PyCQA/flake8/issues/234#issuecomment-812800832)

Reply:
> Strong agree. I don't think `.flake8` support pyproject.toml right now though . But there's other tool we could move already, tox and black probably.

I would recommend to keep `tox` (and `pylint`) separate. Both have quite complex configurations. It's much easier to have them in their individual files. We could however move `isort`, `pytest`, and `mypy`.